@lovettchris It first encounters the missing `prctl` header file
```
/Users/user/Documents/la/AirSim/MavLinkCom/common_utils/ThreadUtils.cpp:9:10: fatal error: 'sys/prctl.h' file not found
#include <sys/prctl.h>
^~~~~~~~~~~~~
1 error generated.
```
Need to remove that as well

Reproduced the error on Travis - https://travis-ci.com/rajat2004/AirSim/builds/144190684

--- a/MavLinkCom/common_utils/ThreadUtils.cpp
+++ b/MavLinkCom/common_utils/ThreadUtils.cpp
@@ -6,7 +6,6 @@
 #include <windows.h> // SetThreadPriority and GetCurrentThread
 #else
 #include <pthread.h>
-#include <sys/prctl.h>
 #endif
 
 using namespace mavlink_utils;

